packages/main/src/ComboBoxGroupItem.ts

```diff
@@ -0,0 +1,64 @@
+import customElement from "@ui5/webcomponents-base/dist/decorators/customElement.js";
+import property from "@ui5/webcomponents-base/dist/decorators/property.js";
+import UI5Element from "@ui5/webcomponents-base/dist/UI5Element.js";
+import type { IComboBoxItem } from "./ComboBox.js";
+import GroupHeaderListItem from "./GroupHeaderListItem.js";
```

Comment:
is this the correct place where to import GroupHeaderListItem? It's used inside ComboBox template.